pkg/orchestrator/jenkins.go

```diff
@@ -14,20 +14,17 @@ import (
 
 type JenkinsConfigProvider struct {
 	client         piperHttp.Client
-	options        piperHttp.ClientOptions
 	apiInformation map[string]interface{}
 }
 
 // InitOrchestratorProvider initializes the Jenkins orchestrator with credentials
 func (j *JenkinsConfigProvider) InitOrchestratorProvider(settings *OrchestratorSettings) {
-	j.client = piperHttp.Client{}
```

Comment:
Since `piperHttp.Client` is a struct and its zero value is an empty struct, there's no need to explicitly create a new instance of `piperHttp.Client`. It is already initialized with a zero-value (empty struct) along with the initialization of `JenkinsConfigProvider` instance.